jupyterlab_gcedetails/src/components/chart_wrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { AreaChart, Area, Tooltip, CartesianGrid, YAxis } from 'recharts';
+
+type ChartProps = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any[];
+  dataKey: string;
+  title: string;
+  titleClass: string;
+  areaChartProps: {};
+  areaProps: {};
+  cartesianGridProps: {};
+  yAxisProps: {};
+};
+
+export const AreaChartWrapper = ({
```

Comment:
We've used the convention of using the `function` keyword to declare functional components (see https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/c1fb81d9c22575255dd372991aeda5df0321aed0/jupyterlab_gcedetails/src/components/details_dialog_body.tsx#L25) as that fits our internal TS style guide.

Also, I think you should be able to just say `props: ChartProps` for the parameter interface.

Reply:
Ok cool, is it better for the object destructuring to happen in the params or in the function body, or just use `props.xxx` ?

Reply:
There's no official guidance in our internal TS style guide, so I'm OK with the destructuring here instead of in the function body.

Reply:
Ok, I've just moved it to the function body to go in line with `DetailsDialogBody`